src/vitabel/timeseries.py

```diff
@@ -225,6 +218,27 @@ def offset(self) -> Timedelta:
         """The offset applied to the time data."""
         return self._offset
     
+    @property
+    def first_entry(self) -> Timestamp | None:
+        '''The time of the earliest entry in time_index, as shown by the info.'''
+        min_time = None
+        if len(self) > 0:
+            min_time = min(self.time_index)
```

Comment:
if the time index is relative, min(...) / max(...) won't be correct: you are not looking for the shortest/longest time span. (actually great question: what even would be correct?)